components/Button/IconButton.d.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
O que acha de usar assim:

```suggestion
import React, {ComponentType} from 'react';
```